satpy/enhancements/__init__.py

```diff
@@ -78,10 +78,10 @@ def apply_enhancement(data, func, exclude=None, separate=False,
             if pass_dask:
                 dims = band_data.dims
                 coords = band_data.coords
-                d_arr = func(band_data.data)
+                d_arr = func(band_data.data, index=idx)
```

Comment:
Doesn't adding an index keyword argument break other enhancement functions that don't expect it?

Reply:
I tried to fix those I could find at least